addons/website_event_questions/tests/test_event_internals.py

```diff
@@ -85,3 +85,24 @@ def test_process_attendees_form(self):
                 (0, 0, {'question_id': self.event_question_2.id, 'value_answer_id': 7}),
                 (0, 0, {'question_id': self.event_question_3.id, 'value_text_box': 'Free Text'})]}
         ])
+
+    def test_question_translation_preservation(self):
```

Comment:
Isn't it possible to concatenate in 'test_event_type_configuration_from_type' so that we test the whole compute (including translations) in a single unit test ?

Reply:
Good idea, that ensures it doesn't pass by simply overwriting the default title & name, instead of actually adding translations.